### PR TITLE
implementation ( Edit Content ) : #30015 Implement Show/Hide Toggle for Hidden Column 

### LINKED ISSUE #30015: Implement Show/Hide Toggle for Hidden Column
## Parent Issue
#30014 

## Task
Implement the show/hide toggle functionality for the hidden column feature.

https://github.com/user-attachments/assets/ff5d7928-47d1-4875-beec-e95a3053e2bd

## Proposed Objective
Same as Parent Issue

## Proposed Priority
Same as Parent Issue

## Acceptance Criteria 
- Create an icon button to toggle the hidden column
- Implement smooth animation for showing/hiding the column
- Ensure the toggle state is preserved across page reloads
- Update any necessary layout components to accommodate the hidden column

## Assumptions & Initiation Needs
- Design assets for the toggle icon are available
- Parent issue is properly set up and accessible

## Quality Assurance Notes & Workarounds
- Verify that the animation is smooth and doesn't cause layout shifts

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* Hide/show the content information. 
* Safe the state based on local storage. 

### Screenshots

https://github.com/user-attachments/assets/b05007a0-5316-4a53-94d7-f2f167651129

### CODE REVIEW

File: core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts [hunk 1]
```diff
@@ -50,36 +58,70 @@ import { DotEditContentService } from '../../services/dot-edit-content.service';
         DotEditContentService,
         MessageService,
         DotEditContentStore
+    ],
```
Comment (oidacra) [inline core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts:61]: We need to plan to move these animations to reuse.

File: core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts [hunk 2]
```diff
@@ -50,36 +58,70 @@ import { DotEditContentService } from '../../services/dot-edit-content.service';
...
+                }
             });
         });
+
+        this.#store.layout$.subscribe((layout) => {
+            this.showSidebar = layout.showSidebar;
+        });
+    }
+
+    toggleSidebar() {
```
Comment (oidacra) [inline core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts:123]: doc

**Review by oidacra — APPROVED** [review]
LGTM

File: core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts (hunk 2 above)
Comment (hmoreras) [reply core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts:128]: done

**Review by valentinogiardino — APPROVED** [review]

**Pull request merged**